portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,22 +52,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         // Number of comments to return to the use
         int numComments;
+        int page;
         try {
             numComments = Integer.parseInt(request.getParameter("numComments"));
+            page = Integer.parseInt(request.getParameter("page"));
         } catch (NumberFormatException e) {
-            numComments = 2;
+            numComments = 5;
```

Comment:
can we log the error here instead of silently ignore it? nit: can we store the magic number into constants?

Reply:
Sure! Yeah I guess I should also alert the user that there was an issue. Also yeah I can move 5 to a const :D